src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -44,11 +44,11 @@ public DependencyContextBuilder WithMainProjectInDepsFile(bool includeMainProjec
             return this;
         }
 
-        public DependencyContextBuilder WithFrameworkReferences(IEnumerable<ReferenceInfo> frameworkReferences)
+        public DependencyContextBuilder WithReferenceAssemblyReferences(IEnumerable<ReferenceInfo> referenceAssemblyReferences)
         {
-            // note: Framework libraries only export compile-time stuff
+            // note: ReferenceAssembly libraries only export compile-time stuff
             // since they assume the runtime library is present already
-            _frameworkReferences = frameworkReferences;
+            _referenceAssemblyReferences = referenceAssemblyReferences;
```

Comment:
Any reason you didn't use _referenceAssemblies?  Duplicated reference term is confusing.